### PR TITLE
Add Contoso Bicycle background info here

### PR DESCRIPTION
**Pull request by ckittel**
The dedicated Contoso Bicycle page is being removed from the main article, and since we reference it, I'm salvaging it and bringing it here into the repo.  Only minor tweaks to it so that it feels like it's "here" instead of "there."

### CODE REVIEW

File: contoso-bicycle/README.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+### Networking team
+
+Make sure that different architectural components can talk to each other in a secure manner. They manage the hub and spoke network topologies and IP space allocation.
+
+### Operations team
+
+Responsible for the infrastructure deployment and day-to-day operations of the Azure environment.
+
+## Business requirements
```
Comment (ferantivero) [inline contoso-bicycle/README.md:45]: aren't we already covering this whole topic from the docs? Could we just cross-link or we do really want this in here.

File: contoso-bicycle/README.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+- Automated deployments: They understand the importance of automation. They build automated processes for all infrastructure so that environments and workloads
+can easily be recreated consistently and at any time.
+
+### Cost optimization
+
+- Cost center: There’s only one line-of-business. So, all costs are billed to a single cost center.
+
+- Budget and alerts: They have certain planned budgets. They want to be alerted when certain thresholds like 50%, 75%, and 90% of the plan has been reached.
+
+## Design and technology choices
```
Comment (ferantivero) [inline contoso-bicycle/README.md:85]: have we considered moving this to main README where we list all the components/technologies in this RI?

File: contoso-bicycle/README.md (hunk 1 above)
Comment (ckittel) [reply contoso-bicycle/README.md:45]: Not as sucinctly -- this list is scattered to the wind now.

File: contoso-bicycle/README.md (hunk 2 above)
Comment (ckittel) [reply contoso-bicycle/README.md:85]: I had not considered that.  I was trying to salvage this before the next docs publish (in about two hours), didn't try to merge it in.  Just took mostly "as is" for now.

**Comment by ckittel** [comment]
Had a convo on the side with @ferantivero -- we'll merge in as-is for now and consider coming back and integrating this in a different way.

**Pull request merged**